tests/dicom/test_coords.py

```diff
@@ -1,97 +1,83 @@
+# Copyright (C) 2018 Matthew Jennings
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
 import json
-from os.path import abspath, dirname
-from os.path import join as pjoin
+
+import pytest
 
 import numpy as np
 
 import pydicom
 
-from pymedphys._dicom.coords import xyz_axes_from_dataset
+import pymedphys
+from pymedphys._data import download
+from pymedphys._dicom import coords
 
-HERE = dirname(abspath(__file__))
-DATA_DIRECTORY = pjoin(HERE, "data", "dose")
 ORIENTATIONS_SUPPORTED = ["FFDL", "FFDR", "FFP", "FFS", "HFDL", "HFDR", "HFP", "HFS"]
 
 
+def get_file_within_data_zip(zip_name, file_name):
+    dose_data_files = pymedphys.zip_data_paths(zip_name)
+    path_match = [path for path in dose_data_files if path.name == file_name]
+
+    assert len(path_match) == 1
+
+    return str(path_match[0])
+
+
 def get_data_file(orientation_key):
-    r"""Read in test DICOM files"""
-    filename = "RD.DICOMORIENT.Dose_{}_empty.dcm".format(orientation_key)
-    return pjoin(DATA_DIRECTORY, filename)
+    return download.get_file_within_data_zip(
+        "dicom_dose_test_data.zip",
+        "RD.DICOMORIENT.Dose_{}_empty.dcm".format(orientation_key),
+    )
 
 
-def run_xyz_function_tests(coord_system, save_new_baseline=False):
+def run_xyz_function_tests(coord_system):
     r"""Run the xyz extraction test sequence for a given
     xyz extraction function"""
 
-    print_ = True
-
-    expected_xyz_filename = "expected_{}_xyz.json".format(coord_system.lower())
+    expected_xyz_path = download.get_file_within_data_zip(
+        "dicom_dose_test_data.zip", "expected_{}_xyz.json".format(coord_system.lower())
+    )
 
-    if save_new_baseline:
-        expected_xyz = {}
-    else:
-        with open(pjoin(DATA_DIRECTORY, expected_xyz_filename)) as fp:
-            expected_xyz = json.load(fp)
+    with open(expected_xyz_path) as fp:
+        expected_xyz = json.load(fp)
 
-        assert set(expected_xyz.keys()) == set(ORIENTATIONS_SUPPORTED)
+    assert set(expected_xyz.keys()) == set(ORIENTATIONS_SUPPORTED)
 
     test_ds_dict = {
         key: pydicom.dcmread(get_data_file(key)) for key in ORIENTATIONS_SUPPORTED
     }
-    print()
-    for orient, dicom in test_ds_dict.items():
-        test_xyz = xyz_axes_from_dataset(dicom, coord_system)
-
-        if save_new_baseline or print_:
```

Comment:
@Matthew-Jennings this was an interesting case. These tests here weren't actually ever running because `print_` was set to `True`...